### PR TITLE
Introduce Operations History Component

### PR DESCRIPTION
**Pull request by cloderic**
Ready to review !

## Tasks ##

- [x] react-infinite less infinitely rendering list
- [x] component displaying already loaded operations
- [x] component working with on demand loading of operations
- [x] test on actual operations lists with regular operations
- [x] test on actual operations lists with irregular operations
- [x] test dynamic changes to the height and row height
- [x] test dynamic changes to the initial operations
- [x] test dynamic changes to 'from'/'to'
- [x] Add icons in row headers
- [x] Test with custom css
- [x] Test searching for a timestamp

### CODE REVIEW

**Comment by cloderic** [comment]
@Giska and @PierreSalaun not all the details are handled yet but the storybook is testable and the code is reviewable.

**Comment by PierreSalaun** [comment]
It would be nice to have the "search" function that allows you to input a timestamp and center the view on the associated context operation as well as highlighting it.
The default behavior would be to focus on the latest context operation when opening the "operation history" tab.
If other operation are added after you open the "operation history" tab, the focus should not change.

**Comment by PierreSalaun** [comment]
When "Loading" is displayed, the width of the view changes.

File: packages/react-craft-ai-operations-history/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# `react-craft-ai-operations-history` #
+
+:construction:
```
[the remark this answers is not in the document]
Comment (cloderic) [reply packages/react-craft-ai-operations-history/README.md]: done !

File: packages/react-craft-ai-decision-tree/package.json [hunk 2]
```diff
@@ -37,12 +37,27 @@
...
-    "babel-core": "6.26.0",
-    "babel-loader": "7.1.2",
-    "eslint": "4.5.0",
-    "webpack": "3.10.0"
+    "babel-core": "6.26.3",
+    "babel-eslint": "10.0.1",
+    "babel-loader": "8.0.4",
+    "babel-plugin-emotion": "9.2.11",
+    "eslint": "5.9.0",
+    "eslint-config-craft-ai": "2.3",
```
[the remark this answers is not in the document]
Comment (cloderic) [reply packages/react-craft-ai-decision-tree/package.json]: will be done in a separate PR

**Pull request merged**